docs/DeveloperGuide.md

```diff
@@ -321,32 +321,214 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `DorManagerPro`, and the 
+**Actor** is the `user` who refers to university dormitory managers 
+unless specified otherwise.
 
-**Use case: Delete a person**
+**Use Case: UC01 - Add a profile**
```

Comment:
I am not very sure about this, but based on the logic for this use case, the user cannot stop until a profile is successfully added.

Should there be an extension "*a. User decides to stop adding a profile" that can happen at any step?

Same for other use cases.